File: py/bank.py
from account import Account
from person import Person

class Bank:
    def __init__(self):
        self.clientData = {}
    def newClient(self, p: Person):
        acc = Account(p)
        self.clientData[p] = acc
        return acc
    def removeClient(self, p: Person):
        self.clientData.pop(p, None)
    def getClientAccount(self, p: Person):
        try:
            acc = self.clientData.get(p)
            if acc is None:
                raise Exception('Person not enrolled!')
            return acc
        except Exception as e:
            print('Error:', e)
            return None

File: java/Bank.java
import java.util.HashMap;

public class Bank {
    HashMap<Person, Account> clientData = new HashMap<>();

    public Account newClient(Person p) {
        Account acc = new Account(p);
        clientData.put(p, acc);
        return acc;
    }

    public void removeClient(Person p) {
        clientData.remove(p);
    }

    public Account getClientAccount(Person p) {
        try {
            Account acc = clientData.get(p);
            if (acc == null) throw new Exception("Person not enrolled!");
            return acc;
        } catch (Exception e) {
            System.out.println("Error: " + e.getMessage());
            return null;
        }
    }
}
